Add button for copy to diff workspace when in read only mode [SATURN-1204]

ehanna4 opened the pull request:
- also fix padding around icons

ehanna4 committed:
Add copy to workspace button when in read only mode, and add padding to buttons

zarsky-broad commented on src/pages/workspaces/workspace/notebooks/NotebookLauncher.js:
```diff
@@ -266,29 +266,30 @@ const PreviewHeader = ({ queryParams, cluster, readOnlyAccess, onCreateCluster,
   return div({ style: { display: 'flex', alignItems: 'center', borderBottom: `2px solid ${colors.dark(0.2)}`, height: '3.5rem' } }, [
     div({ style: { fontSize: 18, fontWeight: 'bold', backgroundColor: colors.dark(0.2), padding: '0 4rem', height: '100%', display: 'flex', alignItems: 'center' } },
       ['PREVIEW (READ-ONLY)']),
+    readOnlyAccess && h(ButtonSecondary, { style: buttonStyle, onClick: () => setExportingNotebook(true) }, [makeMenuIcon('export'), 'Copy to another workspace']),
```
Maybe a ternary?
```js
readOnlyAccess ?
  button :
  cond(
```

zarsky-broad reviewed (commented):
If you're using `makeMenuIcon`, you can get rid of all the `{ style: { paddingRight: 3 } }`

ehanna4 committed:
PR feedback

- remove unused paddingRight style
- use ternary for cleaner logic

ehanna4 committed:
Add ability to rename imported workflow

- Also add tooltip to import button
- todo: validate workflow name isn't already in use in selected workspace

ehanna4 committed:
undo commit (wrong branch)

panentheos reviewed (approved):
Click tested, looks good.
